wg-checkpoint-restore/charter.md

```diff
@@ -0,0 +1,91 @@
+
+# WG Checkpoint Restore Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The Checkpoint/Restore Working Group aims to solve the problem of transparently
+checkpointing and restoring workloads in Kubernetes, a functionality discussed
+for over five years. The group will deliver the design and implementation of
+Checkpoint/Restore functionality in Kubernetes, serving as a central hub for
+community information and discussion. This initiative addresses a wide range of
+problems, including fault tolerance, improved resource utilization, and
+accelerated application startup times.
+
+### In scope
+
+- Identify core Kubernetes checkpoint/restore use cases (e.g., live migration,
+  fault tolerance, debugging, snapshotting) and gather stakeholder requirements.
+- Investigate and propose Kubernetes APIs for checkpoint/restore operations.
+- Work with SIGs for the best integration of checkpoint/restore functionality
+  and APIs.
+- Provide guidance for developers on checkpoint-friendly app design and
+  recommendations for operators on feature management.
+- Work closely with relevant upstream projects (CRI-O, containerd, CRIU, gVisor)
+  for alignment and integration.
+- Revisit the existing implementations to find and remedy possible inefficiencies.
+  One example is the existing checkpoint archive format which has already been
+  identified as being a major source of slowdown.
+
+### Out of scope
+
+- Not focused on general OS-level checkpointing outside Kubernetes
+  pods/containers.
+- Will not dictate internal application checkpointing logic; focuses on
+  Kubernetes platform orchestration of *container/pod state.
+
+## Stakeholders
+
+Stakeholders in this working group span multiple SIGs that own parts of the
+code in core kubernetes components and addons.
+
+- SIG CLI
+- SIG API Machinery
+- SIG Node
+- SIG Scheduling
+- SIG Auth
+- SIG Apps
+
+## Deliverables
+
+The list of deliverables include the following high level features:
+
+- In the early stage, we mainly want to offer a well-defined location for the
```

Comment:
can you elaborate on what "defined location" means here, asking because I find this as a point of confusion in other WG reviews, per example, WG do not own repositories, so if by location we mean "slack channel, mailing list, meeting" then that is fine

Reply:
yes, "well-defined location" here refers to the WG slack channel, mailing list, meetings.